script/upgrades/MU03/MU03.sol

```diff
@@ -120,7 +120,7 @@ contract MU03 is IMentoUpgrade, GovernanceScript {
 
     vm.startBroadcast(Chain.deployerPrivateKey());
     {
-      createProposal(_transactions, "MU03", governance);
+      createProposal(_transactions, "https://google.com", governance);
```

Comment:
I assume this will be updated later when we have a descriptionURL?

Reply:
yes @nvtaveras